src/tensor.cpp

```diff
@@ -99,15 +154,25 @@ int TensorDescriptor::GetSize() const
 std::size_t TensorDescriptor::GetElementSize() const
 {
     assert(lens.size() == strides.size());
-    return std::accumulate(
-        lens.begin(), lens.end(), std::size_t{1}, std::multiplies<std::size_t>());
+    return std::accumulate(lens.begin(), lens.end(), vector_c, std::multiplies<std::size_t>());
 }
 miopenDataType_t TensorDescriptor::GetType() const { return this->type; }
+miopenTensorLayout_t TensorDescriptor::GetLayout_t() const { return this->tensorLayout; }
+int TensorDescriptor::GetVectorLength() const { return this->vector_c; }
```

Comment:
I would suggest syncing the names. Either `GetVectorC` for the method or `vector_length` for the field.

Reply:
Updated in latest commit.